xbmc/cores/VideoPlayer/DVDFileInfo.cpp

```diff
@@ -250,7 +251,8 @@ bool CDVDFileInfo::ExtractThumb(const CFileItem& fileItem,
               aspect = hint.aspect;
             unsigned int nHeight = (unsigned int)((double)nWidth / aspect);
 
-            uint8_t *pOutBuf = (uint8_t*)av_malloc(nWidth * nHeight * 4);
+            int sizeNeeded = av_image_get_buffer_size(AV_PIX_FMT_BGRA, nWidth, nHeight, 64);
```

Comment:
`64` alignment for what? :wink:

Reply:
It's not for the alignment, it's for the additional padding. sws_scale needs 16 bit alignment and sizes need to be 16 bit aligned as well. Shall I reduce it to 16? Just wanted to make sure that any possible intrinsic we get is safe to use.

Reply:
If I set it to 16 instead, it crashes again right away.

Reply:
Okay revisited it. In my testing the , 64 works by chance - what really matters for me seems that the width needs to be dividable by 16.